### PR TITLE
OVAL templating clean-up

### PR DESCRIPTION
**Pull request by mpreisler**
Things are all over the place, as part of this PR I want to move everything into the "new" locations and deduplicate templates as well as static files.

I found a couple templates that apply everywhere but were set RHEL only so I fixed that. Other than that this shouldn't change anything after all the content is built.

I will continue with this work to make sure this is fixed for all products and not just RHELs.

### CODE REVIEW

**Comment by mpreisler** [comment]
Hmm, the way the templating system overwrites files when generating OVAL 5.10 and OVAL 5.11 is just nasty... Why don't we treat OVAL 5.10 and 5.11 as two different languages for checks again? Would make everything much much cleaner...

**Comment by mpreisler** [comment]
Another WTF. We have remediation templates that are marked as RHEL7 only but they reside in the shared folder. What??! https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/templates/template_BASH_accounts_password

**Comment by shawndwells** [comment]
12-MAY will henceforth be known as "Brain Aneurism for @mpreisler Day"

**Comment by mpreisler** [comment]
ROFL: https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/oval/oval_5.11/service_dovecot_disabled.xml

**Comment by mpreisler** [comment]
Up for discussion: Should I remove the "DO NOT EDIT" warnings from the templates? They are confusing when you are actually editing the template. The paths no longer match and I think people are unlikely to edit something in the `build` folder unless they know what they are doing. Thoughts?

Example:
This is a shared template, not RHEL6 only...
```
  <!-- THIS FILE IS GENERATED by RHEL/6/input/checks/templates/create_umask_checks.py.  DO NOT EDIT.  -->
  <!-- INSTEAD OF EDIT THAT TEMPLATE IF NECESSARY. AFTER THAT BE SURE TO REGENERATE ALL CORRESPONDING -->
  <!-- UNCOMMENTED CHECSK FROM file_umask_checks.csv CSV FILE -->
```

**Comment by yuumasato** [comment]
We can remove the warnings, IMO.

**Comment by mpreisler** [comment]
Ready for review. Will do further changes in other PRs.

**Comment by shawndwells** [comment]
Why move files such as ``RHEL7/input/oval/grub2_enable_fips_mode.xml`` to ``RHEL7/input/templates/static/oval/grub2_enale_fips_mode.xml``? They aren't templates.

**Comment by mpreisler** [comment]
> Why move files such as RHEL7/input/oval/grub2_enable_fips_mode.xml to RHEL7/input/templates/static/oval/grub2_enale_fips_mode.xml? They aren't templates.

We are moving everything to the new location chosen in the past by @ybznek . It is a bit confusing but we need to first move everything there before we can rename it. Inventing a third location would be even more confusing.

tl;dr: This is on purpose and is correct.

**Comment by yuumasato** [comment]
Ack.
